stories/GalleryList.stories.js

```diff
@@ -0,0 +1,106 @@
+import GalleryList from "~/components/gallery/List"
+import WpImage from "~/components/WpImage"
+import { data as API } from "~/stories/mock-api.json"
+
+export default {
+    title: "ProgrammingPartners / GalleryList",
+}
+
+export const Default = () => ({
+    data() {
+        return {
+            items: API.directors.children.nodes,
+        }
+    },
+    components: {
+        GalleryList,
+        WpImage,
```

Comment:
This isn't being used, can be removed.

Reply:
Removed. Though removing it before updating updating the packages in `package.json` broke the storybook build.